yarn.lock

```diff
@@ -2367,6 +2367,11 @@ bn.js@^5.1.1:
   resolved "https://registry.yarnpkg.com/bn.js/-/bn.js-5.1.3.tgz#beca005408f642ebebea80b042b4d18d2ac0ee6b"
   integrity sha512-GkTiFpjFtUzU9CbMeJ5iazkCzGL3jrhzerzZIuqLABjbwRaFt33I9tUdSNryIptM+RxDet6OKm2WnLXzW51KsQ==
 
+bn.js@^5.1.1:
```

Comment:
I think you accidentally added this. You shouldn't be adding any yarn file updates unless you've added a package to package.json. The best way to keep yarn up to date is to pull directly from upstream, not your fork, and always run `yarn` before `yarn start:stage` for local development.